packages/application-extension/src/index.tsx

```diff
@@ -104,37 +105,33 @@ const main: JupyterLabPlugin<void> = {
           router.reload();
         }
       }).catch(err => {
-        showDialog({
-          title: 'Build Failed',
-          body: (<pre>{err.message}</pre>)
-        });
+        showDialog({ title: 'Build Failed', body: (<pre>{err.message}</pre>) });
```

Comment:
As long as you are converting things to `showErrorMessage`, how about this one?

Reply:
Good idea, I'll do that.